docs/Modules/bpmn-modules/modeling.md

```diff
@@ -9,4 +9,160 @@ parent: "BPMN Modules"
 grand_parent: "Modules"
 ---
 
-# modeling
\ No newline at end of file
+# modeling
+
+This module activates the BPMN 2.0 modeling features [over the common modeling module](/modules/common-modules/modeling). It means that this module contains every API exported by the common modeling module.
```

Comment:
Not sure what _activates features over the common modeling module_ means. Let's say the module extends the functionality of the existing diagram-js modeling module.

Reply:
Done 👍